docs/setting-up/server/helm.md

```diff
@@ -13,32 +10,78 @@
+- https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html
+
+Kubernetes platforms offer a different set of capabilities. To use PMM in production, you would need backups and, thus storage driver that supports snapshots. Please refer to your Kubernetes and Cloud provider for storage capabilities.
+
+## Locality and Availability
+
+It is not recommended to run the PMM monitoring server on the same system together with monitored database clusters and services.
+
+Please ensure proper locality either by physically separating workloads in Kubernetes clusters or running separate Kubernetes clusters for the databases and monitoring workloads.
+
+Physical separation of workloads can be done with proper Kubernetes nodes configuration, affinity rules, label selections and etc.
+
+Please also ensure the Kubernetes cluster has [high availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) so that in a case of a node failure, the monitoring service will be running and capturing needed data.
+
+PMM, when correctly set up, would be highly available together with Kubernetes proper HA. Please ensure that your network and storage support high availability. In case of Kubernetes node failure - PMM will be rescheduled and available on any other node in a Kubernetes cluster.
+
 ## Use Helm to install PMM server on Kubernetes clusters
 
 !!! note alert alert-primary "Availability"
     This feature is available starting with PMM 2.29.0.
 
 
 !!! summary alert alert-info "Summary"
+    - Setup PMM admin password
     - Install
     - Configuration parameters
-    - PMM admin password
     - PMM environment variables
     - PMM SSL certificates
+    - Backup
     - Upgrade
+    - Restore
     - Uninstall
 
 ---
 
+### Setup PMM admin password
+
+Create Kubernetes secret with PMM admin password:
+```sh
+cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pmm-secret
+type: Opaque
+data:
+# base64 encoded password
+# encode some password: `echo -n "admin" | base64`
+  PMM_ADMIN_PASSWORD: YWRtaW4=
+EOF
+```
+
+To get admin password execute:
+
+```sh
+kubectl get secret pmm-secret -o jsonpath='{.data.PMM_ADMIN_PASSWORD}' | base64 --decode
+```
+
 ### Install
 
 To install the chart with the release name `pmm`:
 
 ```sh
 helm repo add percona https://percona.github.io/percona-helm-charts/
-helm install pmm percona/pmm
+helm install pmm percona/pmm --set secret.create=false --set secret.name=pmm-secret
 ```
-The command deploys PMM on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+The command deploys PMM on the Kubernetes cluster in the default configuration and specified secret. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
```

Comment:
the term **and specified secret** is not clear

Reply:
`with secrets that was created earlier`?